include/openthread/types.h

```diff
@@ -1033,6 +1060,19 @@ typedef struct otSockAddr
     int8_t       mScopeId;  ///< An IPv6 scope identifier.
 } otSockAddr;
 
+/**
+ * This structure represents Thread-specific link information.
```

Comment:
Suggestion: This structure represents link-specific information for messages received from the Thread radio.